src/EnergyPlus/MixedAir.cc

```diff
@@ -1098,6 +1126,12 @@ void GetOutsideAirSysInputs(EnergyPlusData &state)
                     CheckDXCoolingCoilInOASysExists(state, state.dataAirLoop->OutsideAirSys(OASysNum).ComponentName(CompNum));
                 } else if (SELECT_CASE_var == "COILSYSTEM:HEATING:DX") {
                     state.dataAirLoop->OutsideAirSys(OASysNum).ComponentType_Num(CompNum) = static_cast<int>(ComponentType::DXHeatPumpSystem);
+                } else if (SELECT_CASE_var == "COILSYSTEM:COOLING:WATER") {
+                    state.dataAirLoop->OutsideAirSys(OASysNum).ComponentType_Num(CompNum) = static_cast<int>(ComponentType::CoilSystemWater);
+                    state.dataAirLoop->OutsideAirSys(OASysNum).ComponentIndex(CompNum) = CompNum;
+                    UnitarySystems::UnitarySys thisSys;
+                    state.dataAirLoop->OutsideAirSys(OASysNum).compPointer[CompNum] = thisSys.factory(
+                        state, DataHVACGlobals::UnitarySys_AnyCoilType, state.dataAirLoop->OutsideAirSys(OASysNum).ComponentName(CompNum), false, 0);
```

Comment:
Saves the component type and component index and instantiates this coil system as UnitarySystem.